blaze/compute/core.py

```diff
@@ -367,8 +371,14 @@ def swap_resources_into_scope(expr, scope):
 
 
 @dispatch(Expr, dict)
-def compute(expr, d, **kwargs):
-    """ Compute expression against data sources
+def compute(expr, d, return_type='native', **kwargs):
+    """ Compute expression against data sources.
+
+    Args:
+        expr: blaze expression
+        d: data source to compute expression on
+        return_type (optional): type to return data as. Defaults to 'core' which
```

Comment:
For this release `return_type` defaults to `'native'`.  We should also note in the docstring that we'll be changing it to `'core'` in version 0.11.